firmware/user/display/oled.c

```diff
@@ -155,9 +155,9 @@ void ICACHE_FLASH_ATTR clearDisplay(void)
  * @param y2
  * @param c
  */
-void ICACHE_FLASH_ATTR fillDisplayArea(uint8_t x1, uint8_t y1, uint8_t x2, uint8_t y2, color c)
+void ICACHE_FLASH_ATTR fillDisplayArea(int16_t x1, int16_t y1, int16_t x2, int16_t y2, color c)
 {
-    uint8_t x, y;
+    int16_t x, y;
```

Comment:
Do I need to change all my oled calls to int16_t?

Reply:
No, your draw calls still work. Moriarity did some out-of-bounds drawing (like to negative pixels), which this affected